Parkinson/Resources/HTMLContent/15sharing_rsch.html

```diff
@@ -22,11 +22,10 @@
         <p><strong>2- Share with Sage Bionetworks and its partners only:</strong> You can choose to share your study data only with the study team and its partners. The study team includes the sponsor of the research and any other
         researchers or partners named in the consent document. Sharing your data only with Sage Bionetworks means that your data will not be made available to anyone other than those listed in
         the consent document and for the purposes of this study only.</p>
-        <p>If required by law, your data (study data and account information), and the signed consent form may be disclosed to:</p>
+        <p>Your data (study data and account information), and the signed consent form may be disclosed to:</p>
         <ul style="padding-left:20px;">
-            <li>The US National Institute of health, Department of Health and Human Services agencies, Office for Human Research Protection, and other agencies as required,</li>
+            <li>The US Department of Health and Human Services, the Office for Human Research Protection, and other agencies as required by law for verification of the research procedure and data,</li>
             <li>Institutional Review Board who monitors the safety, effectiveness and conduct of the research being conducted,</li>
```

Comment:
Since you removed the (old) last item in the list, you probably want to remove the comma at the end of this (new last) item?